spec/models/rating_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe Rating, type: :model do
+  describe 'db table' do
+    it { is_expected.to have_db_column(:rating).of_type(:integer) }
+    it { is_expected.to have_db_column(:article_id).of_type(:integer) }
+    it { is_expected.to have_db_column(:user_id).of_type(:integer) }
+  end
+
+  describe 'Validation' do
+    it { is_expected.to validate_presence_of :rating }
+    it { is_expected.to validate_presence_of :article_id }
+    it {
+      is_expected.to validate_inclusion_of(:rating)
```

Comment:
We don't need tests for the other models that they now have_many Ratings?

Reply:
Good point, added tests